env.js

```diff
@@ -75,12 +84,14 @@ const PUPPETEER_BINARY_LOCATION = (() => {
     return playwright.chromium.executablePath();
   }
 
-  const browserFetcher = puppeteer.createBrowserFetcher({ product: 'chrome' });
-
-  return browserFetcher.revisionInfo(PUPPETEER_CHROMIUM_REVISION)
-    .executablePath;
+  return chromeFetcher.computeExecutablePath({
```

Comment:
Might check this out to make sure it works in older versions, eg puppeteer 1.20.x